postcode_lookup/templates/includes/postal_votes.html

```diff
@@ -0,0 +1,53 @@
+<{{ section.context.htag_primary }} id="{{toc_id}}">{% trans %}Voting by post{% endtrans %}</{{ section.context.htag_primary }}>
+{% if section.context.can_register %}
+    <{{ section.context.htag_secondary }}>{% trans %}Apply for a postal vote{% endtrans %}</{{ section.context.htag_secondary }}>
+    <p>
+      {% trans %}
+        If you know that you won't be able to get to the polling station on polling day, you can vote by post.
+      {% endtrans %}
+      {% trans date=section.timetable.postal_vote_application_deadline|date_filter %}
+        You have until {{ date }}.
+      {% endtrans %}
+    </p>
+    <p>
+      <a href="https://www.gov.uk/apply-postal-vote" class="o-button o-button--primary o-button--dark o-external-link">
+        {% trans %}Apply for a postal vote{% endtrans %}
+      </a>
+    </p>
+    <p>
+      {% trans %}
+        <a href="https://www.electoralcommission.org.uk/voting-and-elections/ways-vote/apply-vote-post">Read more</a> about voting by post.
+      {% endtrans %}
+    </p>
+{% else %}
+    {%
+        trans
+        date=section.data.date|date_filter,
+        application_deadline=section.timetable.postal_vote_application_deadline|date_filter
+    %}
+    <p>
+      The deadline to apply for a postal vote for the elections taking place on {{ date }} was {{ application_deadline }}.
+      You can <a href="https://www.gov.uk/apply-postal-vote">apply for a postal vote</a> in future elections.
+    </p>
```

Comment:
I'm still on the fence about this sentence.
I feel like this sentence in particular would be good to get feedback from the commission comms team before we send text off for translations.

There are some bits of text I've added like "Apply for a postal vote" where I don't have a translation but I can't see it changing. This sentence feels like the one that is most likely to change.